core/operation/src/main/java/uk/gov/gchq/gaffer/operation/util/AggregatePair.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.operation.util;
+
+import uk.gov.gchq.gaffer.data.element.function.ElementAggregator;
+
+public class AggregatePair {
+    private String[] groupBys = null;
```

Comment:
Please can you rename groupBys to groupBy? There are a few instances of groupBys.